backend/src/test/java/am/alanmiste/spacegeekscorner/sgc/SgcIntegrationTest.java

```diff
@@ -132,4 +133,237 @@ void deleteItem() throws Exception {
+                                                "option_group": "Flat",
+                                                "url": "https://printful-upload.s3-accelerate.amazonaws.com/tmp/cd9b12e890f6e6f57a251d2220cf4749/unisex-staple-t-shirt-white-front-631f189745404.jpg"
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "placement": "back",
+                                        "variant_ids": [
+                                            4012,
+                                            4013,
+                                            4014
+                                        ],
+                                        "mockup_url": "https://printful-upload.s3-accelerate.amazonaws.com/tmp/a709f5eb907903ee43349313d4de463a/unisex-staple-t-shirt-white-back-631f189745b4a.jpg",
+                                        "extra": [
+                                            {
+                                                "title": "Back",
+                                                "option": "Back",
+                                                "option_group": "Flat",
+                                                "url": "https://printful-upload.s3-accelerate.amazonaws.com/tmp/1c5f5cced38486f9a6a5f7334523d072/unisex-staple-t-shirt-white-back-631f189747731.jpg"
+                                            }
+                                        ]
+                                    }
+                                ],
+                                "printfiles": [
+                                    {
+                                        "variant_ids": [
+                                            4012,
+                                            4013,
+                                            4014
+                                        ],
+                                        "placement": "front",
+                                        "url": "https://printful-upload.s3-accelerate.amazonaws.com/tmp/c2df9dc03de596d5db21ea306ac5b654/printfile_front.png"
+                                    },
+                                    {
+                                        "variant_ids": [
+                                            4012,
+                                            4013,
+                                            4014
+                                        ],
+                                        "placement": "back",
+                                        "url": "https://printful-upload.s3-accelerate.amazonaws.com/tmp/70e93e662e28eb934ae450c0e8d13942/printfile_back.png"
+                                    }
+                                ]
+                            },
+                            "extra": []
+                        }
+                        """));
+    }
```

Comment:
Du könntest auch diese ellenlangen json's jeweils in eine Datei auslagern und diese dann aufrufen - das macht alles viel schlanker und lesbarer. 🙃